app/src/main/java/org/gdg/frisbee/android/adapter/EventAdapter.java

```diff
@@ -59,11 +60,24 @@
     private Context mContext;
     private LayoutInflater mInflater;
     private ArrayList<Item> mEvents;
+    private int mDeafultIcon;
 
     public EventAdapter(Context ctx) {
+        this(ctx, R.drawable.icon);
+    }
+    
+    public EventAdapter(Context ctx, @DrawableRes int defaultIcon) {
         mContext = ctx;
         mInflater = LayoutInflater.from(mContext);
         mEvents = new ArrayList<>();
+        mDeafultIcon = defaultIcon;
+    }
+
+    /**
+     * The default icon is used if there is no icon for an event.
+     */
+    public void setDeafultIcon(@DrawableRes int mDeafultIcon) {
```

Comment:
I was going to do like that but it is instantiated and set in
EventListFragment rather then in TaggedEventSeriesFragment. So I didn't
want to change that.

On Tue, Mar 3, 2015, 5:51 PM Friedger Müffke <notifications@github.com>
wrote:

> In app/src/main/java/org/gdg/frisbee/android/adapter/EventAdapter.java
> <https://github.com/gdg-x/frisbee/pull/277#discussion_r25697912>:
>
> >          mContext = ctx;
> >          mInflater = LayoutInflater.from(mContext);
> >          mEvents = new ArrayList<>();
> > +        mDeafultIcon = defaultIcon;
> > +    }
> > +
> > +    /**
> > +     * The default icon is used if there is no icon for an event.
> > +     */
> > +    public void setDeafultIcon(@DrawableRes int mDeafultIcon) {
>
> do we need a setter? better make it final and developers need to create a
> new adapter when showing a new tag
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/277/files#r25697912>.
>